src/utils/api/transaction/lsk.js

```diff
@@ -332,26 +309,23 @@ export const createMultiSignatureTransaction = ({
   } = transactionObject;
 
   const schema = moduleAssetSchemas[moduleAssetId];
-  if (moduleAssetId === MODULE_ASSETS_NAME_ID_MAP.registerMultisignatureGroup) {
-    rawTransaction.mandatoryKeys = mandatoryKeys;
-    rawTransaction.optionalKeys = optionalKeys;
-  }
-
   const transaction = createTransactionObject(rawTransaction, moduleAssetId);
-  const keys = {
-    mandatoryKeys: transaction.asset.mandatoryKeys,
-    optionalKeys: transaction.asset.optionalKeys,
-  };
+  const netId = Buffer.from(networkIdentifier, 'hex');
 
   try {
-    const signedTransaction = transactions.signMultiSignatureTransaction(
-      schema,
-      transaction,
-      Buffer.from(networkIdentifier, 'hex'),
-      passphrase,
-      keys,
-      true,
-    );
+    let signedTransaction;
+
+    if (keys.numberOfSignatures > 0) {
```

Comment:
Would the numberOfSignatures property be defined in accounts api response in case of a non multisignature account?

Reply:
Well spot. No, it will be defined as 0. You can check it here in this call http://service-v5.liskdev.net/api/v2/accounts?publicKey=1849c3a63b7c336fec832fbf394457e86b9c3b2ceef6e1029fd4c0c35b16ed88